### PR TITLE
feat(TYTA-48): setup nodemailer (with MDS emails)

### PR DESCRIPTION
**Pull request by kilianpichard**

### CODE REVIEW

File: src/utils/nodemailer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    host: "smtp-mail.outlook.com", // hostname
+    secureConnection: false, // TLS requires secureConnection to be false
+    port: 587, // port for secure SMTP
+    tls: {
+       ciphers:'SSLv3'
+    },
+    auth: {
+        user: process.env.EMAIL,
+        pass: process.env.EMAIL_PASSWORD,
+    }
```
Comment (remirubis) [inline src/utils/nodemailer.js:15]: It's configured for using our MDS email? :)

**Review by remirubis — COMMENTED** [review]
Just a simple question :)

File: src/utils/nodemailer.js (hunk 1 above)
Comment (kilianpichard) [reply src/utils/nodemailer.js:15]: yes ! outlook protocol for mds emails, you just need to put your credentials in .env

**Review by remirubis — APPROVED** [review]
LGTM!

**Review by BrystoQ — APPROVED** [review]
NiceToMe

**Pull request merged**